exporter/downloadscollector.go

```diff
@@ -60,6 +62,14 @@ func NewDownloadsCollector(ds DownloadsSource) *DownloadsCollector {
 		labels = []string{"info_hash", "name"}
 	)
 
+	// namespace is the top-level namespace for this rTorrent exporter.
```

Comment:
I talked to someone today and realized that it shouldn't be done that way and that the correct way would be to set a label in the prometheus config for the target.

Reason for that is that I run one rtorrent instance for every tracker, so I wanted to have the exporters namespaced too. Which should be done with a label instead of a namespace like I did. Will push change in a bit!